spt/strafestuff.hpp

```diff
@@ -322,7 +322,18 @@ bool Strafe(PlayerData& player, const MovementVars& vars, bool onground, bool ju
 {
 	//DevMsg("[Strafing] ducking = %d\n", (int)ducking);
 	if (jumped && player.Velocity.Length2D() >= vars.Maxspeed * ((ducking || (vars.Maxspeed == 320)) ? 0.1 : 0.5)) {
-		out.Yaw = NormalizeDeg(tas_strafe_yaw.GetFloat() + 180);
+		if tas_strafe_glitchless.GetBool() == 1 {
+			const Vector vel = player.Velocity();
+			out.Yaw = NormalizeDeg(double atan2(double vel.y/vel.x));
```

Comment:
Not sure why you put `double` in here, neither is needed (or correct for that matter).